code/game/objects/items/airlock_painter.dm

```diff
@@ -1,3 +1,8 @@
+/// below these levels trigger the special sprites
+#define PAINTER_MID 0.2
```

Comment:
Considering it looks half-ish full in the icon this should probably be 0.5 and you could change PAINTER_LOW to 0.2 or 0.3